lib/minify-inline.js

```diff
@@ -0,0 +1,35 @@
+const terser = require('terser')
+const fs = require('fs')
+
+module.exports = minifyInline
+
+function minifyInline (filename, opts, cb) {
```

Comment:
Can you add a comment here describing what this function does?

Reply:
Fixed